tools/bin/commands/util/openshift_funcs

```diff
@@ -222,3 +222,42 @@ guess_route() {
     fi
     echo "${project}.${domain#*.}"
 }
+
+create_api_cookie() {
```

Comment:
I wouldn't put it into openshift_funcs as it is minishift specific. This should be mentioned in the help message, too.

Reply:
Also, I guess this is really only used for the `dev` command so happy to keep it there. Only when it needs to be shared it should be moved to the shared lib